src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
```

Comment:
This line does not reflect the class, should mention that it will list the persons in sorted order.